server.js

```diff
@@ -1088,6 +1088,33 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Clojars version integration
+camp.route(/^\/clojars\/v\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var repo = match[1];  // eg, `prismic`.
+  var format = match[2];
+  var apiUrl = 'https://clojars.org/search?q=' + repo + '&format=json';
```

Comment:
Could you escape `repo` please? Something like `encodeURIComponent(repo)` might be enough.